code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -43,7 +44,7 @@
 		else
 			name = initial(name)
 		return
-	else if(I.tool_behaviour == TOOL_WIRECUTTER)
+	else if((I.tool_behaviour == TOOL_WIRECUTTER) && tagged)
```

Comment:
if you're touching this proc, please make this var have a better name

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names